package-lock.json

```diff
@@ -16,7 +16,7 @@
       "integrity": "sha1-63d99gEXI6OxTopywIBcjoZ0a9I=",
       "dev": true,
       "requires": {
-        "mime-types": "2.1.18",
+        "mime-types": "~2.1.18",
```

Comment:
If you do not change the package.json, there is no need to update this file in the PR. Please remove changes to this file.